scripts/testdata/satellite-config.yaml.lock

```diff
@@ -205,6 +205,12 @@ kademlia.operator.wallet: ""
 # validate redundancy scheme configuration
 # metainfo.rs.validate: true
 
+# how long to wait for new observers before starting iteration
+# metainfoloop.coalesce-duration: 5s
+
+# how long to wait between metainfo iterations
+# metainfoloop.interval: 30s
```

Comment:
Is this config still available after the last changes?

Reply:
will update